src/containers/BooksForm.js

```diff
@@ -8,21 +13,48 @@ const BooksForm = () => {
     'Learning',
     'Sci-Fi',
   ];
+  const [state, setState] = useState({ title: '', category: '' });
```

Comment:
- [ ] [REQUIRED] The `category value` should be equal to `Action`

Reply:
Done